lib/packages.cf

```diff
@@ -94,6 +94,28 @@ body package_module yum
     #default_options => {};
 }
 
+body package_module freebsd_ports
+# @brief Define details used when interfacing with the freebsd ports package
+# module.
+#
+# **Note:** Ports are expected to be setup prior to trying to use the packages
+# promise. You may need to ensure that `portsnap extract` has been run.
```

Comment:
Maybe there's a way to check that in `packages.cf` or in the package script itself?